package/gluon-web-admin/files/lib/gluon/config-mode/view/admin/upgrade_confirm.html

```diff
@@ -21,7 +21,7 @@ <h2><%:Upgrade firmware%></h2>
 
 <p>
   <ul>
-    <li>md5sum: <code><%=checksum%></code></li>
+    <li>sha256sum: <code><%=checksum%></code></li>
```

Comment:
Hmm, simply labeling it "Checksum" would look nicer... in that case we should also make this translatable. Well, that can be done in another PR.